examples/tutorials/date_time_charts.py

```diff
@@ -0,0 +1,362 @@
+"""
+Plotting datetime charts
+========================
+
+Plotting datetime data points is handled by :meth:`pygmt.Figure.basemap`.
```

Comment:
I feel that the content of the "Datetime Input Types" section is better introductory sentences for this tutorial.
```suggestion

PyGMT accepts a variety of datetime objects to plot data and create charts.
Aside from the built-in Python datetime object, PyGMT supports input datetime objects
using ``pandas``, ``numpy``, ``xarray`` as well as properly formatted strings.
These data types can be used to plot specific points as well as get
passed into the ``region`` parameter to create a range of the data on an axis.

The following examples will demonstrate how to create plots
using the different datetime objects.
```